tools/test.sh

```diff
@@ -37,7 +37,7 @@ else
 fi
 
 # Where to store the modified app definitions.
-readonly temp_builders_root=$(mktemp -d -t test_run)
+readonly temp_builders_root=$(mktemp -d -t test_run.XXXX)
```

Comment:
I'm not sure why you are adding to this filename.

Reply:
The mktemp tool requires multiple `X` characters so it can generate a unique name in Linux. It does not require this on Mac. I was running this script on my Linux box when I got the warning.